panther_analysis_tool/command/validate.py

```diff
@@ -73,35 +29,17 @@ def run(backend: BackendClient, args: argparse.Namespace) -> Tuple[int, str]:
     buffer.seek(0, 0)
     params = BulkUploadParams(zip_bytes=buffer.read())
 
-    return_code = 0
-    return_str = ""
     try:
         result = backend.bulk_validate(params)
         if result.is_valid():
-            return return_code, f"{BColors.green('validation success')}"
-
-        return_str += "\n-----\n"
-
-        if result.has_error():
-            return_str += f"{BColors.bold('Error')}: {result.error}\n-----\n"
+            return 0, f"{cli_output.success('Validation success')}"
 
-        for issue in result.issues():
-            if issue.path and issue.path != "":
-                return_str += f"{BColors.bold('Path')}: {issue.path}\n"
-
-            if issue.error_message and issue.error_message != "":
-                return_str += f"{BColors.bold('Error')}: {issue.error_message}\n"
-
-            return_str += "-----\n"
-
-        return_code = 1
-        return_str = f"{return_str}\n{BColors.failed('validation failed')}"
+        return 1, cli_output.multipart_error_msg(result, "Validation failed")
     except UnsupportedEndpointError as err:
         logging.debug(err)
-        return 1, BColors.warning("your panther instance does not support this feature")
+        return 1, cli_output.warning("Your Panther instance does not support this feature")
```

Comment:
is there somewhere we could direct the customer to get access or get more information? like some sales page or something to upsell them?

or is this more of a 404 or a mechanism to protect our internal endpoints?

Reply:
That is an interesting question. @nhakmiller Do you have ideas about this?